src/libstore/globals.hh

```diff
@@ -689,19 +690,38 @@ public:
         getDefaultSystemFeatures(),
         "system-features",
         R"(
-          A set of system “features” supported by this machine, e.g. `kvm`.
-          Derivations can express a dependency on such features through the
-          derivation attribute `requiredSystemFeatures`. For example, the
-          attribute
+          A set of system “features” supported by this machine.
+
+          This complements the [`system`](#conf-system) and [`extra-platforms`](#conf-extra-platforms) configuration options and the corresponding [`system`](@docroot@/language/derivations.md#attr-system) attribute on derivations.
+
+          Derivations can require system features in the derivation attribute [`requiredSystemFeatures`](@docroot@/language/advanced-attributes.md#adv-attr-requiredSystemFeatures).
```

Comment:
```suggestion
          Derivations can require system features in the derivation attribute [`requiredSystemFeatures`](@docroot@/language/advanced-attributes.md#adv-attr-requiredSystemFeatures), or indirectly by setting for example [`__contentAddressed`](@docroot@/language/advanced-attributes.html#adv-attr-__contentAddressed).
```

Reply:
That's what I thought at first, but the original sentence isn't truthful.
Also it's not the right place, because this is the `system-features` option documentation, not the documentation of the system features system in general, and this exception is not really about the `system-features` option, but about the interaction.

The items below are all about how the default value is composed. We could change all of this and write a page that's focused on the known features, and then it would be more appropriate to mention it there. https://github.com/NixOS/nix/pull/8706#discussion_r1267935540

Reply:
I guess there's also info about what they represent in the items below, but that does not really relate to Nix's own behavior anyway so I didn't consider that in my comment.
(all the more reason for a separate page perhaps?)